cairo-vm-cli/src/main.rs

```diff
@@ -112,12 +115,21 @@ fn run(args: impl Iterator<Item = String>) -> Result<(), Error> {
             return Err(Error::Cli(error));
         }
     };
-    let trace_enabled = args.trace_file.is_some();
+
+    if args.air_public_input.is_some() && !args.proof_mode {
+        let error = Args::command().error(
+            clap::ErrorKind::ArgumentConflict,
+            "--air_public_input can only be used in proof_mode.",
+        );
+        return Err(Error::Cli(error));
+    }
```

Comment:
This error is not being shown when running the cli, must be due to: #1270

Reply:
Done!